ui/app/components/brave/brave-header/index.scss

```diff
@@ -0,0 +1,100 @@
+.br-toolbar {
```

Comment:
Can we have things in a single top level brave folder with the same duplicated hierarchy?